### PR TITLE
Changed API configuration step in offline documentation

### PR DESCRIPTION
**Pull request by davidcr01**
## Description

Closes: https://github.com/wazuh/wazuh-documentation/issues/7225

The aim of this to change the `localhost` value of the `/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml` file of the Wazuh offline documentation and the Step-by-step documentation of the Wazuh dashboard installation.

This change is done because we do not support the `localhost` value in that file due to https://github.com/wazuh/wazuh-packages/issues/2771. This also implies to remove the `Only for distributed deployments` note.
 
Also, I flxed a typo in the Offline documentation.

Screenshot from 2024-04-25 12-25-05
Screenshot from 2024-04-25 12-25-16

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]

**Review by javimed — CHANGES_REQUESTED** [review]
And what about all the user administration documents? For example password management document or the Google SSO as administrator

- https://github.com/wazuh/wazuh-documentation/blob/bed6341c3c4f0e319756eee1496de0341e4b912a/source/user-manual/user-administration/password-management.rst?plain=1#L243
- https://github.com/wazuh/wazuh-documentation/blob/bed6341c3c4f0e319756eee1496de0341e4b912a/source/user-manual/user-administration/single-sign-on/administrator/google.rst?plain=1#L253

same for LDAP occurrences. Or for example these lines?

- https://github.com/wazuh/wazuh-documentation/blob/bed6341c3c4f0e319756eee1496de0341e4b912a/source/user-manual/wazuh-dashboard/troubleshooting.rst?plain=1#L87
- https://github.com/wazuh/wazuh-documentation/blob/bed6341c3c4f0e319756eee1496de0341e4b912a/source/user-manual/wazuh-dashboard/config-file.rst?plain=1#L57
-

File: source/deployment-options/offline-installation.rst [hunk 1]
```diff
@@ -444,14 +444,14 @@ Installing the Wazuh dashboard
...
-#. **Only for distributed deployments**:  Edit the file ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` and replace the ``url`` value with the IP address or hostname of the Wazuh server master node.
+#.  Edit the file ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` and replace the ``url`` value with the IP address or hostname of the Wazuh server master node.
 
             .. code-block:: yaml
                :emphasize-lines: 3
 
                hosts:
                  - default:
-                     url: https://localhost
+                     url: https://<wazuh_server_ip>
```
[the remark this answers is not in the document]
Comment (javimed) [reply source/deployment-options/offline-installation.rst:454]: We use uppercase and underscores for placeholders.

File: source/installation-guide/wazuh-dashboard/step-by-step.rst
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/installation-guide/wazuh-dashboard/step-by-step.rst:1]: Changed.

File: source/deployment-options/offline-installation.rst
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/deployment-options/offline-installation.rst:1]: Changed.

File: source/installation-guide/wazuh-dashboard/step-by-step.rst [hunk 2]
```diff
@@ -99,20 +99,20 @@ Starting the Wazuh dashboard service
...
-               hosts:
-                 - default:
-                     url: https://localhost
-                     port: 55000
-                     username: wazuh-wui
-                     password: wazuh-wui
-                     run_as: false
+         hosts:
+            - default:
+               url: https://<wazuh_server_ip>
```
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/installation-guide/wazuh-dashboard/step-by-step.rst]: Done

File: source/installation-guide/wazuh-dashboard/step-by-step.rst [hunk 3]
```diff
@@ -99,20 +99,20 @@ Starting the Wazuh dashboard service
       .. include:: /_templates/installations/dashboard/enable_dashboard.rst
```
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/installation-guide/wazuh-dashboard/step-by-step.rst]: Done

File: source/installation-guide/wazuh-dashboard/step-by-step.rst [hunk 4]
```diff
@@ -99,20 +99,20 @@ Starting the Wazuh dashboard service
...
 
       
-      **Only for distributed deployments**  
+
```
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/installation-guide/wazuh-dashboard/step-by-step.rst]: Done

File: source/deployment-options/offline-installation.rst [hunk 5]
```diff
@@ -463,7 +463,7 @@ Installing the Wazuh dashboard
 
 #.  Access the web interface.
 
-    -   URL: *https://<wazuh_server_ip>*
+    -   URL: *https://<wazuh_dashboard_ip>*
```
[the remark this answers is not in the document]
Comment (davidcr01) [reply source/deployment-options/offline-installation.rst]: Done

**Review by javimed — APPROVED** [review]

**Review by teddytpc1 — APPROVED** [review]

**Pull request merged**